packages/react-components/src/lib/cesium-map/data-sources/drawings.data-source.stories.tsx

```diff
@@ -84,8 +86,62 @@ export const Drawings: Story = (args) => {
       >
         Box
       </button>
+      <button
+        style={{ position: 'fixed', top: '80px', left: '20px', zIndex: 1 }}
+        onClick={(): void => {
+          setIsDrawing(false);
+          setDrawPrimitive({
+            type: DrawType.UNKNOWN,
+            // eslint-disable-next-line @typescript-eslint/no-empty-function
+            handler: (drawing: IDrawingEvent) => {},
+          });
+        }}
+      >
+        Stop Draw
+      </button>
+      <button
+        style={{ position: 'fixed', top: '140px', left: '20px', zIndex: 1 }}
+        onClick={(): void => {
+          setIsDrawing(false);
+          setDrawEntities([
+            {
+              coordinates: undefined,
+              name: `${DrawType.BOX.toString()}_KUKU`,
+              id: 'KUKU',
+              type: DrawType.BOX,
+              geojson: {
+                type: 'FeatureCollection',
+                features: [
+                  {
+                    type: 'Feature',
+                    properties: {
+                      type: BboxCorner.BOTTOM_LEFT,
```

Comment:
is this specified in wiki or some readme file?
how to use explanation?

Reply:
Unfortunately, no wiki. STORY is a interactive wiki